editor/quick_open.cpp

```diff
@@ -113,12 +113,14 @@ void EditorQuickOpen::_sbox_input(const Ref<InputEvent> &p_ie) {
 
 float EditorQuickOpen::_path_cmp(String search, String path) const {
 
+	int pos = -1;
 	if (search == path) {
 		return 1.2f;
 	}
-	if (path.findn(search) != -1) {
-		return 1.1f;
-	}
+
+	else if ((pos = path.rfindn(search)) != -1)
```

Comment:
This `else` should be right after the closing bracket of the previous `if`.

Reply:
https://github.com/godotengine/godot/commit/38a8f65779b03469e33652ab40f3bbdf8edd6133
I have fixed the formatting @YeldhamDev you can now review it. So is the PR approvable now?